cla_backend/libs/eligibility_calculator/cfe_civil/income.py

```diff
@@ -10,20 +10,28 @@
     "other_income": "friends_or_family",
 }
 
+_CFE_INCOME_KEY = "regular_transactions"
+
+
+def has_income_key(dict):
+    return _CFE_INCOME_KEY in dict
+
 
 def translate_income(income_data):
     regular_transactions = []
 
     for income_category in INCOME_CATEGORY_TO_REGULAR_TRANSACTION:
         if hasattr(income_data, income_category):
-            amount_pence = getattr(income_data, income_category, None)
-            if amount_pence:
-                regular_transactions.append(
-                    {
-                        "category": INCOME_CATEGORY_TO_REGULAR_TRANSACTION[income_category],
-                        "operation": "credit",
-                        "frequency": "monthly",
-                        "amount": pence_to_pounds(amount_pence)
-                    }
-                )
-    return {"regular_transactions": regular_transactions} if regular_transactions else {}
+            amount_pence = getattr(income_data, income_category)
+            regular_transactions.append(
```

Comment:
@starswan We are not going to use regular_transactions with amount=0 anywhere. What if we only add non-zero regular_transactions only?